wsdlgo/encoder.go

```diff
@@ -23,7 +23,8 @@ import (
 	"strings"
 	"text/template"
 
-	"github.com/fiorix/wsdl2go/wsdl"
+	"github.com/Kernle32DLL/wsdl2go/wsdl"
+	"github.com/iancoleman/strcase"
```

Comment:
Can we avoid this new dependency? iirc we currently have zero external dependencies.